packages/reflector/src/com/split/metatable/policies/MetadataAccessPolicy.ts

```diff
@@ -39,29 +17,152 @@ export enum PrimitiveMetadataAccessPolicy {
  * - PARAMETER_IN_METHOD - the decorated will be proceeded if it is defined for parameter in instance or static method
  * - PARAMETER - the decorated will be proceeded if it is defined for parameter in instance or static method or in constructor
  * - ALL - the decorated will be proceeded for all class members
+ *
+ * @public
+ * @interface
  */
-export enum MetadataAccessPolicy {
-    NONE = PrimitiveMetadataAccessPolicy.NONE,
+export interface PrimitiveMetadataAccessPolicyType {
+    NONE: number;
+    CONSTRUCTOR: number
+    INST_PROPERTY: number;
+    INST_ACCESSOR: number;
+    INST_METHOD: number;
```

Comment:
How about using `INSTANCE` instead of `INST`? It's just four more chars :) While reading it makes things more obvious.

Reply:
Especially while reading the readme.